base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -910,6 +929,26 @@ where
         Ok(())
     }
 
+    /// Go through all completed transactions that have not yet been broadcast and broadcast all of them to the base
+    /// node followed by mempool requests to confirm that they have been received
+    async fn broadcast_all_completed_transactions_to_mempool(
+        &mut self,
+        broadcast_timeout_futures: &mut FuturesUnordered<BoxFuture<'static, TxId>>,
+    ) -> Result<(), TransactionServiceError>
+    {
+        let completed_txs = self.db.get_completed_transactions().await?;
+        for completed_tx in completed_txs.values() {
+            if completed_tx.status == TransactionStatus::Completed {
+                error!("Completed Txs: {:?}", completed_tx.tx_id);
```

Comment:
missing: `target :LOG_TARGET` here?

Reply:
Urg, was a temp debugging message should be removed.